tcms/bugs/tests/test_views.py

```diff
@@ -68,3 +80,60 @@ def test_reopen_a_closed_bug(self):
         self.assertRedirects(response, redirect_url)
         bug.refresh_from_db()
         self.assertTrue(bug.status)
+
+
+class TestNewBug(LoggedInTestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        user_should_have_perm(cls.tester, 'bugs.add_bug')
+
+        cls.url = reverse('bugs-new')
+
+        cls.summary = 'A shiny new bug!'
+        cls.product = ProductFactory()
+        cls.version = VersionFactory(product=cls.product)
+        cls.build = BuildFactory(product=cls.product)
+        cls.post_data = {
+            'summary': cls.summary,
+            'reporter': cls.tester.pk,
+            'product': cls.product.pk,
+            'version': cls.version.pk,
+            'build': cls.version.pk,
+        }
+
+    def test_get_view(self):
+        response = self.client.get(self.url)
+
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.context['page_title'], 'New bug')
```

Comment:
Page titles are translated and this causes test failures when tested with languages other than English. Don't hard code the value, use gettext_lazy to translate. See how other tests do it.